src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -307,6 +307,7 @@
         color: $ibexa-color-dark;
         fill: $ibexa-color-dark;
         padding: calculateRem(11px) calculateRem(12px);
+        pointer-events: auto;
```

Comment:
pointer-events: auto is default value
shouldnt we delete all of the pointer-events: none values?